RecFCCeeCalorimeter/src/components/CalibrateCaloClusters.cpp

```diff
@@ -169,9 +169,24 @@ StatusCode CalibrateCaloClusters::finalize()
 {
   if (m_ortSession)
     delete m_ortSession;
+
   if (m_ortEnv)
     delete m_ortEnv;
 
+  for (std::size_t i = 0; i < m_input_names.size(); i++) {
+    if (m_input_names.at(i)) {
+      delete m_input_names.at(i);
+    }
+  }
+  m_input_names.resize(0);
```

Comment:
```suggestion
  for (auto& name : m_input_names) {
    delete name;
  }
```
This can be shortened to this, because:
- we loop over all elements, so a range-based for loop more clearly expresses that extent.
- `delete` will happily accept a `nullptr` (and do the right thing with it)
- the `resize` doesn't have to happen, since the whole algorithm and with it the `vector` will be destroyed very soon after this. At that point it will contain a bunch of pointers that have already been cleaned up, so it will also do the right thing automatically.